app/src/Components/Event/EventDetails.tsx

```diff
@@ -96,6 +100,9 @@ const EventDetails: FC<EventDetailsProps> = ({ event }) => {
             {event.description}
           </Markdown>
         </EventDescription>
+        {/* if there is an event speaker show this. Otherwise nothing.
```

Comment:
Please can you remove these two lines of commented out code (commented code should almost always be removed when you come to PR stage)